services/ui-src/src/components/IdleTimerWrapper.tsx

```diff
@@ -64,7 +64,7 @@ const IdleTimerWrapper = () => {
      * this depends on promptTimeout and logoutTimeout
      * this is to ensure that the idleTimer has the most recent values for the times
      */
-    if (isAuthenticated) {
+    if (isAuthenticated && !isLoggedInAsDeveloper) {
```

Comment:
Woot!

Reply:
crazy, though, I think this line is what is failing the codeclimate diff-coverage test...

Reply:
yeah....hmmmm......all i can think of is OVERRIDE! :)